repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/GenericLayoutEntity.java

```diff
@@ -110,4 +111,8 @@ public UserRoleEntity getUserRole() {
     public void setUserRole(UserRoleEntity userRole) {
         this.userRole = userRole;
     }
+
+    public Optional<String> fetchShowCondition() {
```

Comment:
I assume we can't call it `getShowConditon`. Maybe we could rename to just `showCondition`? fetch is a big confusing

Reply:
done